src/Input/InputCollection.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { useContent, WithContentProps } from "../Content/Content";
+import { AiOutlineClose, AiOutlinePlus } from "react-icons/ai";
+
+const StyledInputContainer = styled.div`
+  background-color: ${props => props.theme.colors.primary.lightgrey};
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  margin: 15px 0;
+`;
+
+const StyledOuter = styled.div`
+  margin-bottom: 8px;
+`;
+
+const StyledInner = styled.div`
+  display: flex;
+`;
+
+const StyledInput = styled.input`
+  border: 1px solid;
+  border-color: ${props => props.theme.colors.primary.grey};
+  border-radius: 7px;
+  width: 95%;
+  height: 40px;
+  padding-left: 16px;
+  font-size: 1rem;
+  &::placeholder {
+    color: ${props => props.theme.colors.primary.mediumgrey};
+  }
+  &:focus {
+    outline: none;
+    border-color: ${props => props.theme.colors.primary.dark};
+  }
+`;
+
+const StyledAddButton = styled.button`
+  display: flex;
+  border: 1px solid;
+  border-color: ${props => props.theme.colors.primary.grey};
+  background-color: ${props => props.theme.colors.primary.lightgrey};
+  width: 95%;
+  height: 40px;
+  text-align: left;
+  font-size: 16px;
+  color: ${props => props.theme.colors.primary.mediumgrey};
+  border-radius: 7px;
+  &:focus {
+    outline: none;
+    border-color: ${props => props.theme.colors.primary.dark};
+  }
+`;
+
+const StyledRemoveButton = styled.button`
+  width: 5%;
+  border: none;
+  font-size: 20px;
+  color: #b4b7bd;
+  background-color: ${props => props.theme.colors.primary.lightgrey};
+  &:focus {
+    outline: none;
+    border: 1px solid ${props => props.theme.colors.primary.dark};
+  }
+`;
+
+const StyledAiOutlinePlus = styled(AiOutlinePlus)`
+  margin-right: 15px;
+`;
+
+const StyledError = styled.div`
+  color: red;
+  margin: 0px;
```

Comment:
Can we add a bit more styling here? 
- No need for font-family because that is all done when you use it in the app
- Let's add some padding though to aline it with the inputs